modules/nodes-clusters-cgroups-2.adoc

```diff
@@ -35,9 +35,14 @@ Currently, disabling CPU load balancing is not supported by cgroup v2. As a resu
 * You have a running {product-title} cluster that uses version 4.12 or later.
 * You are logged in to the cluster as a user with administrative privileges.
 
+[NOTE]
```

Comment:
I was hoping for a note in the OSP file directly, so our OSP NFV users know they have to read this module.

Reply:
@EmilienM I'm sorry about that. I misunderstood. I'll make that update right away.

Reply:
No big deal! Thanks for working on that!